CMakeLists.txt

```diff
@@ -14,10 +14,14 @@ find_package(Freetype REQUIRED)
 find_package(Threads REQUIRED)
 find_package(X11 REQUIRED)
 find_package(ZLIB REQUIRED)
-pkg_search_module(GTK REQUIRED gtk+-3.0)
+pkg_search_module(GTK gtk+-3.0)
 
-set(CURSES_NEED_NCURSES TRUE)
-find_package(Curses REQUIRED)
+set(CURSES_NEED_NCURSES FALSE)
+find_package(Curses)
+
+if(GTK_FOUND)
+   target_compile_definitions(graphics PRIVATE HAVE_GTK)
+endif()
```

Comment:
I think you'll have to move this down to the very bottom of the file as otherwise you'll get
```
CMake Error at CMakeLists.txt:23 (target_compile_definitions):
  Cannot specify compile definitions for target "graphics" which is not built
  by this project.
```

Reply:
Strange, I don't get this error (I'm on CMake 3.20.5). I do however get an error in `g_src/renderer_curses.cpp` claiming that `mvwaddstr` is not defined. I'll move it down anyway.